src/API/Google/AdsCampaign.php

```diff
@@ -71,10 +72,12 @@ class AdsCampaign implements ContainerAwareInterface, OptionsAwareInterface {
 	 *
 	 * @param GoogleAdsClient   $client
 	 * @param AdsCampaignBudget $budget
+	 * @param GoogleHelper      $google_helper
 	 */
-	public function __construct( GoogleAdsClient $client, AdsCampaignBudget $budget ) {
-		$this->client = $client;
-		$this->budget = $budget;
+	public function __construct( GoogleAdsClient $client, AdsCampaignBudget $budget, GoogleHelper $google_helper ) {
+		$this->client        = $client;
+		$this->budget        = $budget;
+		$this->google_helper = $google_helper;
```

Comment:
Just missing to define this as a class parameter (like is done for the other two), otherwise it will default to public, but we want it to be protected.

Reply:
👀 👀 Thanks! Added in 0811070.